charts/secrets-store-csi-driver/README.md

```diff
@@ -19,67 +19,73 @@ $ helm install csi-secrets-store secrets-store-csi-driver/secrets-store-csi-driv
+| `linux.image.tag`                       | Linux image tag                                                                                                                   | `v0.0.22`                                               |
+| `linux.affinity`                        | Linux affinity                                                                                                                    | `key: type; operator: NotIn; values: [virtual-kubelet]` |
+| `linux.driver.resources`                | The resource request/limits for the linux secrets-store container image                                                           | `limits: 200m CPU, 200Mi; requests: 50m CPU, 100Mi`     |
+| `linux.enabled`                         | Install secrets store csi driver on linux nodes                                                                                   | true                                                    |
+| `linux.kubeletRootDir`                  | Configure the kubelet root dir                                                                                                    | `/var/lib/kubelet`                                      |
+| `linux.providersDir`                    | Configure the providers root dir                                                                                                  | `/etc/kubernetes/secrets-store-csi-providers`           |
+| `linux.nodeSelector`                    | Node Selector for the daemonset on linux nodes                                                                                    | `{}`                                                    |
+| `linux.tolerations`                     | Tolerations for the daemonset on linux nodes                                                                                      | `[]`                                                    |
+| `linux.metricsAddr`                     | The address the metric endpoint binds to                                                                                          | `:8095`                                                 |
+| `linux.registrarImage.repository`       | Linux node-driver-registrar image repository                                                                                      | `k8s.gcr.io/sig-storage/csi-node-driver-registrar`      |
+| `linux.registrarImage.pullPolicy`       | Linux node-driver-registrar image pull policy                                                                                     | `IfNotPresent`                                          |
+| `linux.registrarImage.tag`              | Linux node-driver-registrar image tag                                                                                             | `v2.1.0`                                                |
```

Comment:
Could you also update this image to `v2.2.0` for linux and windows in `charts` and `manifest_staging/charts`?

Reply:
For follow up we should update the manifest_staging readme with the new image versions when we update the images in the `values.yaml`.

Reply:
Ah so this was built off of `manifest_staging/charts/secrets-store-csi-driver/README.md`, and we just forgot to update the values on the PRs that updated `manifest_staging...values.yaml`?

Reply:
That's right!